QuizApp/QuizApp/UI/Leaderboard/LeaderboardTableViewCell.swift

```diff
@@ -0,0 +1,77 @@
+import UIKit
+
+class LeaderboardTableViewCell: UITableViewCell {
+
+    static let reuseIdentifier = String(describing: LeaderboardTableViewCell.self)
+
+    private var rankLabel: UILabel!
+    private var nameLabel: UILabel!
+    private var pointsLabel: UILabel!
+
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+
+        createViews()
+        styleViews()
+        defineLayoutForViews()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    func set(rank: Int, name: String, points: Int) {
+        rankLabel.text = "\(rank+1)."
```

Comment:
Logic `(rank + 1)` should be decided somewhere else. This component just draws what it is told.